app/src/openxr/cpp/OpenXRHelpers.h

```diff
@@ -18,6 +18,18 @@ namespace crow {
   const vrb::Vector kAverageHeight(0.0f, 1.7f, 0.0f);
 #endif
 
+const unsigned long kPicoHandTrackingRuntimeVersion = XR_MAKE_VERSION(3, 0, 1);
```

Comment:
The problem is that devices with OS < 5.7.1 do also declare that same version for the runtime. I've just verified it with a Pico 4E using a several months old version (5.6). I'm afraid you'd have to use the system version as we did in the past.

Reply:
Right, I'm working on migrating the checks.

Reply:
Done.